app-emulation/docker/files/early-docker.service

```diff
@@ -1,13 +1,17 @@
 [Unit]
 Description=Early Docker Application Container Engine
 Documentation=http://docs.docker.com
-After=early-docker.socket
-Requires=early-docker.socket
+After=containerd.service early-docker.socket
+Requires=containerd.service early-docker.socket
```

Comment:
I think I was able to observe this working - @mischief when you're around, can you talk me through the issues you had? It's not impossible that I misread the results of my (very manual, ad-hoc) testing, or didn't test the right cases.